docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -19,6 +19,16 @@ default:: undefined
 A dictionary holding the https://crossplane.io/docs/v0.13/introduction/providers.html[Crossplane providers] to be installed.
 The keys of the dict are used as names for the Provider objects.
 
+== `monitoring.enabled`
```

Comment:
```suggestion
=== `enabled`
```

So far I used third level headings for this. I guess we never agreed on how to do this. 🤷🏼‍♂️